config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,27 @@
+periodics:
+- name: serviceaccount-admission-controller-migration
```

Comment:
not sure if there's a convention for naming these. I'd tend to prefix with sig-auth. Thoughts, @BenTheElder ?

Reply:
prepended with sig-auth